package.json

```diff
@@ -53,12 +53,14 @@
     "import-all.macro": "^2.0.3",
     "jest": "^24.0.0",
     "jest-each": "^0.5.0",
+    "jest-get-type": "24.0.0",
     "jest-watch-typeahead": "^0.2.0",
     "lint-staged": "^6.0.0",
     "prettier": "^1.7.4",
     "pretty-format": "^22.1.0"
   },
-  "dependencies": {
+  "peerDependencies": {
+    "@types/jest": "^22.0.0",
     "expect": "^24.1.0",
```

Comment:
Neither `expect` nor `jest-matcher-utils` should need to be deps - what are they used for?

Reply:
For `expect`, several predicates have `import { equals } from '../../utils';`, which in turn has
```
import { equals } from 'expect/build/jasmineUtils';
...
export { equals };
```
```
$ grep -l "from '../../utils'" src/matchers/*/*.js
src/matchers/toBeArrayOfSize/index.js
src/matchers/toBeEmpty/predicate.js
src/matchers/toBeOneOf/predicate.js
src/matchers/toContainAllEntries/predicate.js
src/matchers/toContainAllKeys/predicate.js
src/matchers/toContainAllValues/predicate.js
src/matchers/toContainAnyEntries/predicate.js
src/matchers/toContainAnyValues/predicate.js
src/matchers/toContainEntries/predicate.js
src/matchers/toContainEntry/predicate.js
src/matchers/toContainValue/predicate.js
src/matchers/toContainValues/predicate.js
src/matchers/toIncludeAllMembers/predicate.js
src/matchers/toIncludeAnyMembers/predicate.js
src/matchers/toIncludeSameMembers/predicate.js
```
I know there was some talk about whether this was needed, but I didn't honestly understand why.

For `jest-matcher-utils`, most matchers import that
```
$ grep -l "from 'jest-matcher-utils'" src/matchers/*/*.js
src/matchers/toBeAfter/index.js
src/matchers/toBeArray/index.js
src/matchers/toBeArrayOfSize/index.js
src/matchers/toBeBefore/index.js
src/matchers/toBeBoolean/index.js
src/matchers/toBeDate/index.js
src/matchers/toBeEmpty/index.js
src/matchers/toBeEven/index.js
src/matchers/toBeExtensible/index.js
src/matchers/toBeFalse/index.js
src/matchers/toBeFinite/index.js
src/matchers/toBeFrozen/index.js
src/matchers/toBeFunction/index.js
src/matchers/toBeHexadecimal/index.js
src/matchers/toBeNaN/index.js
src/matchers/toBeNegative/index.js
src/matchers/toBeNil/index.js
src/matchers/toBeNumber/index.js
src/matchers/toBeObject/index.js
src/matchers/toBeOdd/index.js
src/matchers/toBeOneOf/index.js
src/matchers/toBePositive/index.js
src/matchers/toBeSealed/index.js
src/matchers/toBeString/index.js
src/matchers/toBeTrue/index.js
src/matchers/toBeValidDate/index.js
src/matchers/toBeWithin/index.js
src/matchers/toContainAllEntries/index.js
src/matchers/toContainAllKeys/index.js
src/matchers/toContainAllValues/index.js
src/matchers/toContainAnyEntries/index.js
src/matchers/toContainAnyKeys/index.js
src/matchers/toContainAnyValues/index.js
src/matchers/toContainEntries/index.js
src/matchers/toContainEntry/index.js
src/matchers/toContainKey/index.js
src/matchers/toContainKeys/index.js
src/matchers/toContainValue/index.js
src/matchers/toContainValues/index.js
src/matchers/toEndWith/index.js
src/matchers/toEqualCaseInsensitive/index.js
src/matchers/toHaveBeenCalledAfter/index.js
src/matchers/toHaveBeenCalledBefore/index.js
src/matchers/toInclude/index.js
src/matchers/toIncludeAllMembers/index.js
src/matchers/toIncludeAnyMembers/index.js
src/matchers/toIncludeMultiple/index.js
src/matchers/toIncludeRepeated/index.js
src/matchers/toIncludeSameMembers/index.js
src/matchers/toReject/index.js
src/matchers/toResolve/index.js
src/matchers/toSatisfy/index.js
src/matchers/toSatisfyAll/index.js
src/matchers/toStartWith/index.js
src/matchers/toThrowWithMessage/index.js
```

Correct me if I've misunderstood something here.

Reply:
All matchers are injected with `this.equals` (and `this.utils`): https://jestjs.io/docs/expect#thisequalsa-b